apps_list_utils.py

```diff
@@ -0,0 +1,131 @@
+import sys
+
+resource_requested = sys.argv[1]
+
+MACROS_FILE_INIT = "#ifndef MACROS_H\n#define MACROS_H\n"
+
+MACROS_FILE_END = "#endif"
+
+MACROS = "#define {} {}\n"
+
+MACRO_FILE_PATH = "macros.h"
```

Comment:
macros.h shouldn't go in the root  of the repo, but should be created inside the output folder. Take a look at the theme engine for more information.